packages/react-storage/src/components/StorageBrowser/views/Controls/Table.tsx

```diff
@@ -189,23 +189,21 @@ const sortFns = {
 const LocationDetailViewColumnEmptyHeaderMap = ['download'];
 
 export const LocationDetailViewTable = ({
+  items,
   handleDroppedFiles,
-  onNavigate,
-  range,
+  handleLocationItemClick
 }: {
+  items: LocationItemData[];
   handleDroppedFiles: (files: File[]) => void;
-  onNavigate?: (location: LocationData) => void;
-  range: [start: number, end: number];
+  handleLocationItemClick: (location: LocationData) => void;
```

Comment:
Probably not really in scope or even relevant to your PR but we should probably stick to the `onX` convention for props to be consistent. But it shouldn't be a concern here as we will replace this table anyway

Reply:
Yeah, we have an existing `handleDroppedFiles` prop here, so I wanted to stick to that convention. But in general we should use `onX` 👍🏽